engine/src/test/java/com/datatorrent/stram/StramMiniClusterTest.java

```diff
@@ -227,6 +242,7 @@ public void testSetupShutdown() throws Exception
   private LogicalPlan createDAG(LogicalPlanConfiguration lpc) throws Exception
   {
     LogicalPlan dag = new LogicalPlan();
+    addExtraTestJar(dag);
```

Comment:
Found a better way. (I think).. I've parameterized the  createDAG method whether to add extra test  jars or not.